python/tvm/_ffi/libinfo.py

```diff
@@ -98,6 +98,10 @@ def find_lib_path(name=None, search_path=None, optional=False):
     use_runtime = os.environ.get("TVM_USE_RUNTIME_LIB", False)
     dll_path = get_dll_directories()
 
+    manual_dll_path = os.getenv("TVM_DLL_PATH")
```

Comment:
i think we should say LIB not DLL, since DLL is meaningful only on windows. also, how does this differ from TVM_LIBRARY_PATH?